pkgs/tools/security/crowdsec/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub, installShellFiles, jq, fetchpatch }:
+
+buildGoModule rec {
+  pname = "crowdsec";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "crowdsecurity";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-NnFSOxYOz6Rins2QbpOPHgrdQe35RPraGGtkVoB8gaQ=";
+    # populate values that require us to use git. By doing this in postFetch we
+    # can delete .git afterwards and maintain better reproducibility of the src.
+    leaveDotGit = true;
+    postFetch = ''
+      cd "$out"
+      git rev-parse HEAD > $out/COMMIT
+      # 0000-00-00_00:00:00
+      git log --date=format:'%Y-%m-%d_%H:%M:%S' -1 --pretty=%ad > $out/SOURCE_DATE_EPOCH
+      find "$out" -name .git -print0 | xargs -0 rm -rf
+    '';
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/06kellyjac/crowdsec/commit/07f2b64f63ed339bc7f7664192452060929ccd2b.patch";
+      sha256 = "sha256-bTfJt3ZfHUsLRPGF6mHnaEsGNoMCvWS0sA4Ylw/o5ao=";
+    })
+  ];
```

Comment:
Thank you a lot for taking the time to package this!
Maybe we should not tie this patch to your Github account : having it in plain text in the folder would be a good alternative maybe ?

Reply:
It's part of this PR:

https://github.com/crowdsecurity/crowdsec/pull/1302

previously I've pulled a patch using https://github.com/crowdsecurity/crowdsec/pull/1302.patch but I was asked to replace it with the commit. I should have put a comment pointing to the PR

I only commit patches locally if they're not going to make it upstream and are nixpkgs specific

Reply:
I can swap it to be the commit referenced https://github.com/crowdsec/crowdsec/commit/07f2b64f63ed339bc7f7664192452060929ccd2b.patch

Reply:
Sounds great! Let me know when you need crashtest subjects!

Reply:
Will do. I at least need to add some bouncers and maybe add a nixos module for configuration

Reply:
That PR seems to have been merged, do we no longer need the patch after this?